sensor_msgs/NavSatFix gps

bool status

float64 northing
float64 easting
float64 down

string utm_zone

float64 v_n
float64 v_e
float64 v_d

